pandas/core/groupby/generic.py

```diff
@@ -583,7 +583,9 @@ def transform(self, func, *args, **kwargs):
             if not (func in base.transform_kernel_whitelist):
                 msg = "'{func}' is not a valid function name for transform(name)"
                 raise ValueError(msg.format(func=func))
-            if func in base.cythonized_kernels:
+
+            # transformation are added as well since they are broadcasted already
+            if func in base.cythonized_kernels or func in base.transformation_kernels:
```

Comment:
Instead of this can't you just add cumcount to the transformation list? This somewhat blurs the line between `cythonized_kernels` which I think describe performance ops and `transformation_kernels` which describe output shape